src/NuGet.Core/NuGet.ProjectModel/Utf8JsonStreamReader.cs

```diff
@@ -250,7 +253,7 @@ internal IReadOnlyList<string> ReadStringArrayAsReadOnlyListFromArrayStart()
 
             while (Read() && _reader.TokenType != JsonTokenType.EndArray)
             {
-                string value = _reader.ReadTokenAsString();
+                string value = _reader.ReadTokenAsString(); // TODO NK - why is this one null accepting?
```

Comment:
This PR isn't marked as draft, and according to our coding guidelines, we don't do TODOs in our code without a linked github issue.

Reply:
My bad, removed.